controllers/vault.go

```diff
@@ -75,7 +75,7 @@ func kubernetesLogin(ctx context.Context, cl *vault.Client, role string) (string
 	}
 
 	// Authenticate with Vault using Kubernetes JWT.
-	secret, err := cl.Auth.KubernetesLogin(ctx, schema.KubernetesLoginRequest{Jwt: strings.TrimSpace(string(saToken)), Role: role})
+	secret, err := cl.Auth.KubernetesLogin(ctx, schema.KubernetesLoginRequest{Jwt: strings.TrimSpace(string(saToken)), Role: role}, vault.WithMountPath(mountPath))
```

Comment:
Does withMountPath work if mountPath is empty?

Reply:
yes it defaults to "kubernetes"